vite.config.ts

```diff
@@ -62,7 +62,7 @@ export default defineConfig(({ mode }) => {
       commonjsOptions: {
         transformMixedEsModules: true
       },
-      sourcemap: true
+      sourcemap: false
```

Comment:
Removing the sourcemaps is a tradeoff that we need to take at this moment, as building the distributable in vercel is quite impossible.